src/main/java/com/epam/preprod/sirenko/ClothingFactoryImpl.java

```diff
@@ -0,0 +1,17 @@
+package com.epam.preprod.sirenko;
+
+import com.epam.preprod.sirenko.entity.Clothing;
+import com.epam.preprod.sirenko.entity.Product;
+
+public class ClothingFactoryImpl implements CreateProductFactory {
+	
+	@Override
+	public Product createProduct(Strategy strategy) {
```

Comment:
You shoudnt pass stratagy to factory as method param.Do it in constructor during creation

Reply:
done